src/pages/spawn/spawnHost/SpawnHostCard.tsx

```diff
@@ -11,12 +12,16 @@ import { MyHost } from "types/spawn";
 
 interface SpawnHostCardProps {
   host: MyHost;
+  sendEvent: ReturnType<typeof useSpawnAnalytics>["sendEvent"];
 }
 
-export const SpawnHostCard: React.VFC<SpawnHostCardProps> = ({ host }) => (
+export const SpawnHostCard: React.VFC<SpawnHostCardProps> = ({
+  host,
+  sendEvent,
```

Comment:
Rather than passing down the `sendEvent` function, I wonder if it would make sense to just redo the `useSpawnAnalytics()` call here. That way you wouldn't have to edit the corresponding story to have an unused / irrelevant prop

Reply:
Done